dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/com.ibm.websphere.appserver.jaxws-2.3.feature

```diff
@@ -5,6 +5,29 @@ singleton=true
 IBM-App-ForceRestart: uninstall, \
  install
 IBM-ShortName: jaxws-2.3
+IBM-API-Package: \
```

Comment:
Had to add API-Packages as none of these were being exposed to the application class path